src/login/login.component.tsx

```diff
@@ -3,8 +3,10 @@ import { performLogin } from "./login.resource";
 import { always } from "kremling";
 import styles from "./login.component.css";
 import { getCurrentUser } from "@openmrs/esm-api";
+import { getConfig } from "@openmrs/esm-module-config";
 
 export default function Login(props: LoginProps) {
+  const config = getConfig("@openmrs/esm-login");
```

Comment:
Are you planning on switching this one over to useModuleConfig()? Since getConfig is async?

Also, like we discussed in the other pr I think it would be nice and very possible to not require passing in the current module name as an argument if we render some react context with its value already there.

Reply:
I figured I'd refactor it to work with the import map config file once I actually have [that](https://github.com/openmrs/openmrs-esm-module-config/pull/5) working. I'm fine with putting this PR on hold behind that one, however, if you think that would make more sense.

Reply:
That's fine -- if we do so, though, changing `getConfig()` to return a promise will bring down openmrs-spa.org's login page until we update esm-login to expect getConfig to return a promise.

That's probably fine, though, as long as we don't keep openmrs-spa.org's login page down for more than a few mins (since it's the dev environment for all the people working on MFs)